BaselineOfIceberg.package/.filetree

```diff
@@ -1,5 +1,4 @@
 {
-	"separateMethodMetaAndSource" : false,
 	"noMethodMetaData" : true,
-	"useCypressPropertiesFile" : true
-}
\ No newline at end of file
+	"separateMethodMetaAndSource" : false,
+	"useCypressPropertiesFile" : true }
```

Comment:
It could be cool if:
- The exporter could not rewrite metadata without changes
- The exporter could not change the order of metadata between two export.

I am the only one to be bothered by this?